scripts/test_env.py

```diff
@@ -143,10 +146,25 @@ def __init__(self, args: argparse.Namespace, nth: int):
                 logging.error(f"Some of the config files are missing. \
                                           Please provide them in the `{target}` directory")
                 sys.exit(1)
+
+            sign_command = [self.out_dir / "kagami", "crypto", "sign-transaction", "--chain-id", CHAIN_ID,
```

Comment:
Choice of `kagami crypto sign-transaction --genesis-file ... ...` looks a little bit out of place. I would more like `kagami genesis sign ...`, because it is closely related to genesis.

Reply:
Initially I thought that it would be better placed within crypto but after all these changes I moved it to genesis